addons/web/static/tests/webclient/actions/embedded_action.test.js

```diff
@@ -501,11 +518,21 @@ test("execute a regular action from an embedded action", async () => {
 });
 
 test("custom embedded action loaded first", async () => {
+    // set embedded action 4 in first
+    patchWithCleanup(user, {
+        get settings() {
+            return {
+                id: user.userId,
+                embedded_actions_config:
+                    '{"4+1":{"embeddedActionsVisibility":{"104":true},"embeddedVisibility":false,"embeddedActionsOrder":[104, false]}}',
+            };
+        },
+    });
     await mountWithCleanup(WebClient);
-    browser.localStorage.setItem(`orderEmbedded4++${user.userId}`, JSON.stringify([104, false])); // set embedded action 4 in first
     await getService("action").doActionButton({
         name: 4,
         type: "action",
+        resId: 1,
```

Comment:
Doesn't make sense:
action 4 has list view as first viewMode, and there is an expect after that says we end up on that list view.
Passing resId here doesn't make sense